dev/devicelab/lib/tasks/hot_mode_tests.dart

```diff
@@ -10,30 +12,96 @@ import '../framework/adb.dart';
 import '../framework/framework.dart';
 import '../framework/utils.dart';
 
+final Directory _editedFlutterGalleryDir = dir(path.join(Directory.systemTemp.path, 'edited_flutter_gallery'));
+final Directory flutterGalleryDir = dir(path.join(flutterDirectory.path, 'examples/flutter_gallery'));
+
+void _copy(Directory source, Directory target) {
```

Comment:
Could you please move this into `lib/framework/utils.dart`? There's already a `copy` function, so many rename this to `recursiveCopy`?

Reply:
Done